notifications/tests/tests.py

```diff
@@ -457,6 +458,31 @@ def test_unread_list_api_mark_as_read(self):  # pylint: disable=invalid-name
                          self.message_count - 2*num_requested)
         self.assertEqual(len(data['unread_list']), num_requested)
 
+    def test_unread_all_objects(self):
+        """
+        Test notification with all objects (actor, target, action_object)
+        Test that object URLs are in the output.
+        """
+        self.login('to', 'pwd')
+        Notification.objects.filter(recipient=self.to_user).mark_all_as_read()
+        to_customer = Customer.objects.create(name="to_customer")
+        action_customer = Customer.objects.create(name="action_customer")
+        from_customer = Customer.objects.create(name="from_customer")
+        target_object = TargetObject.objects.create(name="target_object")
+        notify.send(from_customer, recipient=self.to_user, verb='commented', action_object=action_customer,
+                    target=target_object)
+        response = self.client.get(reverse('notifications:live_unread_notification_list'))
```

Comment:
I recommend maintaining consistency in the quoting style, some strings are quoted with single quotes, while other are quoted with double quotes. In most repositories I contribute/maintain we have the black formatter handling these format issues but here I guess we have to resort for manual checks unless we introduce black formatting enforcement in another PR.

Reply:
fixed